<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaceFolderDlg</class>
 <widget class="QDialog" name="PlaceFolderDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>395</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>395</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>395</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New Folder</string>
  </property>
  <widget class="QTabWidget" name="propertyTab">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>430</width>
     <height>271</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tabDescription">
    <attribute name="title">
     <string>Description</string>
    </attribute>
    <widget class="QPlainTextEdit" name="textDescription">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>405</width>
       <height>215</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>100</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Description:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabView">
    <attribute name="title">
     <string>View</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>301</width>
       <height>136</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Latitude:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineViewLatitude"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Longitude:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineViewLongitude"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Range:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineViewRange"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Heading:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineViewHeading"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Tilt:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineViewTilt"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="snapshotViewButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>210</y>
       <width>150</width>
       <height>24</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string>Snapshot current view</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="resetButton">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>210</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLineEdit" name="lineName">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>370</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>45</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Name: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>365</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>365</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineName</tabstop>
  <tabstop>propertyTab</tabstop>
  <tabstop>textDescription</tabstop>
  <tabstop>lineViewLatitude</tabstop>
  <tabstop>lineViewLongitude</tabstop>
  <tabstop>lineViewRange</tabstop>
  <tabstop>lineViewHeading</tabstop>
  <tabstop>lineViewTilt</tabstop>
  <tabstop>snapshotViewButton</tabstop>
  <tabstop>resetButton</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
